### ISSUE TITLE
not able to get data on battletag/player

### ISSUE THREAD

**Issue by nocodeinput** [SunDwarf/OWAPI#169]
## Issue
No competitive data for this profile even tho there should be. 
Overbuff: https://www.overbuff.com/players/pc/knut-22248?mode=competitive

This user has no data on quick play, which might be causing issue sin the scrape proccess?

## Links that cause the issue
`https://owapi.net/api/v3/u/knut-22248/blob`

**Comment by SunDwarf** [comment]
...how the hell did that acc manage to get competitive but not quickplay?

Regardless, I'll take a look and try and fix it.

**Comment by nabijaczleweli** [comment]
Well it's possible they played Arcade till L25, right?

**Comment by SunDwarf** [comment]
lol the last time I played overwatch was last may I wouldn't know

**Comment by SunDwarf** [comment]
The edge case to break all edge cases.

The old method of ensuring a user had competitive stats was to see how many groups of stats they had. When looking up stats, it would count the number of groups, and if there was 1 it was only quickplay and returned immediately; if there was 2 there was guaranteed to be competitive stats.

However, that didn't work for users without quickplay stats, which ideally wouldn't even be possible. I fixed it by explicitly trying to find the div with quickplay or competitive, then scanning downwards for the stat group box. 

The issue has been fixed and deployed now.
